package.json

```diff
@@ -50,7 +50,7 @@
     "cheerio": "1.0.0-rc.2",
     "xmldom": "0.1.27",
     "logops": "2.1.0",
-    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
+    "iotagent-node-lib": "git://github.com/flopezag/iotagent-node-lib.git#feature/cluster-support",
```

Comment:
Just a remark: to rollback this change just before merging into master.

Reply:
Fixed c5be33f